Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,120 @@
+            Assert.AreEqual("cierra@example.com", webTablePage.GetCellText(TestSettings.cEmail));
+            Assert.AreEqual("10000", webTablePage.GetCellText(TestSettings.cSalary));
+            Assert.AreEqual("Insurance", webTablePage.GetCellText(TestSettings.cDepartament));
+
+            GenericPages.WebTablePage.ClickAddButton();
+
+            GenericPages.RegFormPage.EnterFirstName();
+            GenericPages.RegFormPage.EnterLastName();
+            GenericPages.RegFormPage.EnterUserEmail();
+            GenericPages.RegFormPage.EnterAge();
+            GenericPages.RegFormPage.EnterSalary();
+            GenericPages.RegFormPage.EnterDepartment();
+
+            GenericPages.WebTablePage.ClickSubmitButton();
+
+            Assert.AreEqual($"{TestSettings.firstName}"+$"{TestSettings.lastName}"+$"{TestSettings.age}"+$"{TestSettings.userEmail}"+$"{TestSettings.salary}"+$"{TestSettings.department}", webTablePage.ReturnTextResult());
+
+            GenericPages.WebTablePage.ClickEditButton();
+            GenericPages.RegFormPage.ClearSalary();
+            GenericPages.RegFormPage.EnterEditedSalary();
+            GenericPages.WebTablePage.ClickSubmitButton();
+
+            Assert.AreEqual($"{TestSettings.firstName}"+$"{TestSettings.lastName}"+$"{TestSettings.age}"+$"{TestSettings.userEmail}"+$"{TestSettings.editedUserSalary}"+$"{TestSettings.department}", webTablePage.ReturnTextResult());
+
+            GenericPages.WebTablePage.ClickDeleteButton();
+        }
+
+        [Test]
+        public void ForthButtonsTest()
+        {
+            var buttonPage = new ButtonPage();
+
+            GenericPages.ButtonPage.GoToButtonsPage();
+
+            GenericPages.ButtonPage.DoubleClick();
+            Assert.AreEqual("You have done a double click", buttonPage.TextResult("doubleClickMessage"));
+
+            GenericPages.ButtonPage.RightClick();
+            Assert.AreEqual("You have done a right click", buttonPage.TextResult("rightClickMessage"));
+
+            GenericPages.ButtonPage.DynamicClick();
+            Assert.AreEqual("You have done a dynamic click", buttonPage.TextResult("dynamicClickMessage"));
+        }
+
+        [Test]
+        public void FifthLinksTest()
+        {
+            var linksPage = new LinksPage();
+
+            GenericPages.LinksPage.GoToLinksPage();
+
+            GenericPages.LinksPage.ClickSimpleLink();
+
+            WebDriverExtensions.SwitchToTheLastWindow();
+            Assert.AreEqual(Constants.newUrl, linksPage.GetUrlAttribute());
+
+            WebDriverExtensions.SwitchToTheFirstWindow();
+
+            GenericPages.LinksPage.ClickCreated();
+            Assert.AreEqual("Link has responded with staus 201 and status text Created", linksPage.TextResult("linkResponse"));
+
+            GenericPages.LinksPage.ClickNoContent();
+            var noContentButtonResultText = Driver.GetWebDriverWait().Until(drv => linksPage.TextResult("linkResponse") == Constants.noContent);
```

Comment:
all bool variables should start from is, ex. isNoContent....

Reply:
Fixed